tensorflow_serving/model_servers/BUILD

```diff
@@ -141,6 +141,11 @@ py_test(
         "//tensorflow_serving/servables/tensorflow/testdata:half_plus_two/00000123/export.data-00000-of-00001",
         "//tensorflow_serving/servables/tensorflow/testdata:half_plus_two/00000123/export.index",
         "//tensorflow_serving/servables/tensorflow/testdata:half_plus_two/00000123/export.meta",
+        "//tensorflow_serving/servables/tensorflow/testdata:half_plus_two2/00000123/export.data-00000-of-00001",
```

Comment:
How about making it half_plus_three, and making it output 0.5*x + 3 ... so we can be sure the model server is running the right model (half-plus-two versus half-plus-three).